dockerfiles/ubuntu/obam/README.md

```diff
@@ -10,7 +10,8 @@ This section defines the step-by-step instructions to build an [Ubuntu](https://
   + Host the downloaded artifacts locally or on a remote location.
   > The hosted locations of artifacts will be passed as the build arguments when building the Docker image.<br>
   > 1. **WSO2_OB_Accelerator_DIST_URL** - Accelerator location
-  > 2.  **WSO2_OB_CERTS_URL** - Certificate zip archive location
+  > 2. **WSO2_OB_CERTS_URL** - Certificate zip archive location
```

Comment:
why do we need both `WSO2_OB_CERTS_URL` and `WSO2_OB_SERVER_CERTS_URL` here

Reply:
`WSO2_OB_CERTS_URL` is for the root and issuer certs. `WSO2_OB_CERTS_URL` is for the new keystores which i generated with new sever certs. should I change the `WSO2_OB_SERVER_CERTS_URL` as the `WSO2_OB_KEYSTORES_URL` ?

Reply:
+1

Reply:
Also root and issuer are from external entities, so it is wrong to say `WSO2_OB_CERTS_URL`.
Lets rename it to `OB_TRUSTED_CERTS`. wdyt @DivyaPremanantha

Reply:
I think OB_TRUSTED_CERTS is better

Reply:
After discussing with divya offline following names were decided.

WSO2_OB_CERTS_URL -> OB_TRUSTED_CERTS_URL
WSO2_OB_SERVER_CERTS_URL -> WSO2_OB_KEYSTORES_URL

Reply:
changed it. https://github.com/wso2/docker-open-banking/pull/148/commits/046cd9c650f919568f07cef69d37317151cd5d2e